utils/scripts/install-el8.sh

```diff
@@ -33,6 +34,7 @@ dnf --nodocs install \
     golang \
     graphviz \
     help2man \
+    hdf5-devel \
```

Comment:
We always require hdf5-devel now?

Reply:
I was trying to build all of the RPMs in daos.spec and the serialization one requires it